indra/assemblers/pysb_assembler.py

```diff
@@ -107,8 +108,11 @@ def add_site_states(self, site, states):
             if state not in self.site_states[site]:
                 self.site_states[site].append(state)
 
-    def add_activating_modification(self, activity_pattern):
-        self.activating_mods.append(activity_pattern)
+    def add_active_form(self, activity_pattern, is_active):
```

Comment:
This method name is confusing as it can also be used to add _inactive_ forms via `is_active=False`. Perhaps `add_activity_form` or something like that?